cdci_data_analysis/analysis/queries.py

```diff
@@ -545,6 +542,9 @@ def get_query_products(self,instrument,job,run_asynch,query_type='Real',logger=N
             else:
                 debug_message=''
 
+            if os.environ.get('DISPATCHER_DEBUG', 'yes') == 'yes':
```

Comment:
do we still need this? I think maybe yes?

Reply:
It is set here:
[https://github.com/oda-hub/dispatcher-app/blob/6330adeb99832ec23d05943e705badb87badd10a/tests/test_configurer.py#L5](https://github.com/oda-hub/dispatcher-app/blob/6330adeb99832ec23d05943e705badb87badd10a/tests/test_configurer.py#L5)

and used here:
[https://github.com/oda-hub/dispatcher-app/blob/6330adeb99832ec23d05943e705badb87badd10a/tests/test_configurer.py#L32](https://github.com/oda-hub/dispatcher-app/blob/6330adeb99832ec23d05943e705badb87badd10a/tests/test_configurer.py#L32)

Reply:
So I would say that is is good to have it for testing purposes

Reply:
absolutely. My concern here was this change is present in this PR, I thought it is already there. But it's ok, it's fine